python/docs/source/reference/pyspark.pandas/frame.rst

```diff
@@ -314,9 +314,9 @@ specific plotting methods of the form ``DataFrame.plot.<kind>``.
    DataFrame.hist
    DataFrame.kde
 
-Koalas-specific
----------------
-``DataFrame.pandas_on_spark`` provides Koalas-specific features that exists only in Koalas.
+Pandas APIs on Spark-specific
+-----------------------------
+``DataFrame.pandas_on_spark`` provides pandas APIs on Spark-specific features that exists only in pandas APIs on Spark.
```

Comment:
@itholic, I think you should rewrite some wordings instead of just switching "Koalas" to "pandas APIs on Spark". For example, "pandas APIs on Spark-specific" doesn't look much making sense. You could also use "pandas-on-Spark" or "pandas on Spark" casually.